tests/unit/test_snap.py

```diff
@@ -299,6 +299,64 @@ def test_rpc_6(self, mock_etree, mock_parse):
             c = mock_log.call_args_list[0]
             self.assertNotEqual(c[0][0].find("ERROR occurred"), -1)
 
+
```

Comment:
Add test case for scenario where there are multiple test files and duplicate command/rpc are distributed among the files